### PR TITLE
Add Version.equals() method

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This commit adds` Version.equals()` method.

This was required while working on a draft for #659.

ONE-vscode-DCO-1.0-Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

File: src/Backend/Version.ts [hunk 1]
```diff
@@ -32,7 +32,13 @@ class Version {
     return ret;
   }
 
-  // TODO: Add operator <, <=, ==, >, >=
+  equals(ver: Version): Boolean {
```
Comment (seanshpark) [inline src/Backend/Version.ts:35]: (optional) I'm not sure about this but there is `Boolean` Object and `boolean` primitive type. I think return type would be `boolean`

Comment (yunjayh) [reply src/Backend/Version.ts:35]: FYI) https://stackoverflow.com/questions/64443288/what-is-the-difference-between-boolean-and-boolean-in-typescript

Comment (hyunsik-yoon) [reply src/Backend/Version.ts]: Thanks!

**Review by seanshpark — APPROVED** [review]
LGTM

File: src/Backend/Version.ts (hunk 1 above)
Comment (hyunsik-yoon) [reply src/Backend/Version.ts]: Interesting.
```ts
if (new Boolean(false)) {
    console.log('run only when if true');
}
```

Comment (seanshpark) [reply src/Backend/Version.ts]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean
```
var x = Boolean(expression);     // use this...
var x = !!(expression);          // ...or this
var x = new Boolean(expression); // don't use this!
```

**Review by yunjayh — APPROVED** [review]

**Pull request merged**